contracts/near/src/state.rs

```diff
@@ -0,0 +1,155 @@
+use {
+    near_sdk::{
+        borsh::{
+            self,
+            BorshDeserialize,
+            BorshSerialize,
+        },
+        serde::{
+            Deserialize,
+            Serialize,
+        },
+    },
+    p2w_sdk::{
+        Identifier,
+        PriceAttestation,
+        PriceStatus,
+    },
+    structs::Chain as WormholeChain,
+};
+
+/// The PriceFeed structure is stored in the contract under a Price Feed Identifier.
+///
+/// This structure matches the layout of the PriceFeed structure in other Pyth receiver contracts
+/// but uses types that are native to NEAR.
+#[derive(BorshDeserialize, BorshSerialize, Serialize)]
```

Comment:
We have simplified the price feed struct, removing "price status" and other obselete fields - see https://github.com/pyth-network/pyth-crosschain/blob/main/aptos/contracts/sources/price_feed.move